src/entry.js

```diff
@@ -4,14 +4,14 @@ import SlideDeck from './index'
 
 const mod = require(DOC_FILENAME)
 const slides = mod.default
-const { theme, components, Provider } = mod
+const { theme, highlight, components, Provider } = mod
 
 export default class App extends React.Component {
   render () {
     return (
       <SlideDeck
         slides={slides}
-        theme={theme}
+        theme={{...theme, highlight}}
```

Comment:
Ah, I see what's happening here now... Do you think it's even worth supporting this outside the theme? I'm starting to think that options like `components` and `Provider` might also make more sense to be in the same `theme` object 🤔

Reply:
Maybe we can group them. What would the advantage be?

Reply:
This is totally a non-blocker, just thinking, but the idea being to simplify the API a bit and make themes a little more portable by including most of the reusable configuration anyone might want to add